build.sbt

```diff
@@ -0,0 +1,157 @@
+ThisBuild / version := "0.8.0-SNAPSHOT"
+ThisBuild / organizationHomepage := Some(url("http://lakhin.com/"))
+ThisBuild / scmInfo := Some(
+  ScmInfo(
+    url("https://github.com/your-account/your-project"),
+    "scm:git@github.com:Eliah-Lakhin/papa-carlo.git"
+  )
+)
+ThisBuild / developers := List(
+  Developer(
+    id = "Eliah-Lakhin",
+    name = "Ilya Lakhin",
+    email = "eliah.lakhin@gmail.com",
+    url = url("http://lakhin.com/")
+  )
+)
+ThisBuild / description := "Constructor of incremental parsers in Scala using PEG grammars"
+ThisBuild / licenses := List("Apache 2.0" -> new URL("http://www.apache.org/licenses/LICENSE-2.0.txt"))
+ThisBuild / startYear := Some(2013)
+ThisBuild / homepage := Some(url("http://lakhin.com/projects/papa-carlo/"))
+ThisBuild / pomIncludeRepository := { _ => false }
+ThisBuild / publishTo := {
+  val nexus = "https://oss.sonatype.org/"
+  if (isSnapshot.value)
+    Some("snapshots" at nexus + "content/repositories/snapshots")
+  else
+    Some("releases"  at nexus + "service/local/staging/deploy/maven2")
+}
+ThisBuild / publishMavenStyle := true
+
+// these versions must be static (no "latest.integration")
+ThisBuild / scalaVersion := "2.13.0"
+ThisBuild / crossScalaVersions := Seq("2.13.0")
+
+import sbt._
+import sbt.Keys._
+
+//addSbtPlugin("com.jsuereth" % "sbt-pgp" % "latest.integration")
+import com.jsuereth.sbtpgp._
+
+//addSbtPlugin("org.scala-js" % "sbt-scalajs" % "latest.integration")
+enablePlugins(ScalaJSPlugin)
+
+// "If you are using a Build.scala definition, import the following"
+import org.scalajs.sbtplugin.ScalaJSPlugin
+import org.scalajs.sbtplugin.ScalaJSPlugin.autoImport._
+
+
+
+  // used by JVM and JSDemo
+  lazy val baseSettings = Seq(
+    scalacOptions += "-unchecked",
+    //sourceDirectory := (sourceDirectory in PapaCarlo).value, // CIRCULAR
+  )
+
+
+
+  lazy val jsSettings =
+    Defaults.coreDefaultSettings ++
+    baseSettings ++
+    ScalaJSPlugin.projectSettings ++
+    Seq(
+      libraryDependencies += "com.lihaoyi" %%% "utest" % "latest.integration" % "test",
+      testFrameworks += new TestFramework("utest.runner.Framework"),
+
+      //sourceDirectory := (sourceDirectory in PapaCarlo).value, // CIRCULAR
+
+      //jsEnvInput := "not working"
+/*
+[error] Cannot determine `jsEnvInput`: Linking result does not have a module named `main`. Set jsEnvInput manually?
+[error] Full report:
+[error] Report(
+[error]   publicModules = [
+[error]
+[error]   ],
+[error] )
+*/
+
+      excludeFilter in unmanagedSources := "test"
+    )
+
+
+
+  lazy val PapaCarlo: sbt.Project = Project(
+    id = "root",
+    //id = "main", // did not help
+    base = file(".")
+  )
+  .enablePlugins(ScalaJSPlugin)
+  .aggregate(JVM, JSDemo)
+
+
+
+  lazy val JVM = Project(
+    id = "jvm",
+    base = file("./jvm/")
+  )
+
+  .settings(sbt.Defaults.coreDefaultSettings: _*)
+
+  .settings(SbtPgp.projectSettings: _*) // projectSettings globalSettings
```

Comment:
This should probably be removed as well. Perhaps replaced by an `enablePlugins(SbtPgp)`, but you'll have to check their readme.